translate.sh

```diff
@@ -109,8 +109,12 @@ copy_translations () {
         for lang in ${LANGUAGES[*]}
         do
             folder_name=${extension//[-]/_}
-            name=ckanext-$extension 
-            cp /usr/lib/adx/build/translations/$lang/$name.po /usr/lib/adx/$name/ckanext/$folder_name/i18n/$lang/LC_MESSAGES/
+            name=ckanext-$extension
+            ext_name=$name
+            if [ $name = "ckanext-ytp-request" ]; then
+                ext_name=ckanext-ytp_request
```

Comment:
The ytp-request is a tricky extensions regarding the paths. The explicit exception here is not a bad idea.
I don't think it should be handled in such a deep place inside a function though.

I suggest rewriting this with using already existing mechanism of replacing `-` with `_` as in the `folder_name` above.
Also please think a little bit about better names as we have here folder_name, name and ext_name. I'm sure they can be named better and in an easier to understand way.

Reply:
I will fix this